src/flask_app.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Provide the FlaskApp class to represent the Flask application."""
+import json
+
+from charm_state import CharmState
+from consts import FLASK_ENV_CONFIG_PREFIX
+
+
+class FlaskApp:  # pylint: disable=too-few-public-methods
```

Comment:
For this, and any other disable, should we add a comment explaining why the disable is required in this case?

Reply:
I think the name `too-few-public-methods` is enough self-explanatory here?